pacemaker.spec.in

```diff
@@ -169,7 +170,7 @@ It will run scripts at initialization, when machines go up or down,
 when related resources fail and can be configured to periodically check
 resource health.
 
-Available rpmbuild rebuild options:
+Available rpmbuild rebuild options (not necessarily applicable/with effect):
```

Comment:
I don't think the additional wording helps the user understand anything; to me, it can only add doubts. The existing text makes it clear that these are the available options; it doesn't make any claims about what they do or don't do. If someone is advanced enough to do an rpm rebuild, they can investigate which options they need. The additional text doesn't help them in any way.